test/integration/targets/vyos_user/tests/cli/basic.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Create user
+  vyos_user:
+    name: netend
```

Comment:
I wonder if it's worth having more obvious names, is their a length limit?
ansibletest1
ansibletest2
ansibletest3

etc